devops/README.md

```diff
@@ -29,23 +29,23 @@ CI pipeline stages:
 
 ## CD pipeline
 
-CD pipeline is described by [Jenkinsfile.cd](aws-codebuild/Jenkinsfile.cd). It uses [Jenkins shared library](https://github.com/evernym/jenkins-shared/tree/aws-codebuild) API as well and generates the following artifacts:
+CD pipeline is described by [Jenkinsfile.cd](aws-codebuild/Jenkinsfile.cd). It uses [Jenkins shared library](https://github.com/sovrin/jenkins-shared/tree/aws-codebuild) API as well and generates the following artifacts:
 - debian package for ubuntu `xenial`
```

Comment:
Should be https://github.com/sovrin-foundation/jenkins-shared/tree/aws-codebuild